lib/dry/logger/entry.rb

```diff
@@ -150,14 +139,29 @@ def filter(filter)
       # @api private
       def build_payload(payload)
         if exception?
-          {exception: exception.class,
-           message: exception.message,
-           backtrace: exception.backtrace || EMPTY_BACKTRACE,
-           **payload}
+          {exception: exception, **payload}
         else
           payload
         end
       end
+
+      # @since 1.0.0
+      # @api private
+      def exception_hash
+        return EMPTY_HASH unless exception?
+
+        {
+          exception: exception.class,
+          message: exception.message,
+          backtrace: exception.backtrace || EMPTY_ARRAY
+        }
+      end
+
+      # @since 1.0.0
+      # @api private
+      def utc_time
+        @utc_time ||= time.utc.iso8601
```

Comment:
use `Time#getutc`, `Time#utc` modifies the receiver https://ruby-doc.org/core-3.1.2/Time.html#method-i-utc